configs.example.ts

```diff
@@ -1,45 +0,0 @@
-// Step 1: Remove the `.example` from this file name so it is called `configs.ts` or copy the contents of the `configs.example.ts` file into a new `configs.ts` file
```

Comment:
I would keep this file, as I am seeing you still kept a section to host without Docker